packages/code-studio/src/styleguide/Dialog.tsx

```diff
@@ -54,7 +54,7 @@ class Dialog extends Component<unknown, DialogState> {
   }
 
   handleUpdateCheckboxMap(
-    checkBoxMap: Map<string, Map<string, boolean>>
+    checkBoxMap: Map<string, boolean | Map<string, boolean>>
```

Comment:
Not a big deal since it's just the styleguide, but looks like there's a type for this in this file. `HierarchicalCheckboxValueMap`. Just use that instead